cylc/flow/scheduler.py

```diff
@@ -1052,7 +1052,8 @@ def load_flow_file(self, is_reload=False):
             mem_log_func=self.profiler.log_memory,
             output_fname=os.path.join(
                 self.workflow_run_dir, 'log', 'flow-config',
-                workflow_files.WorkflowFiles.FLOW_FILE + '.processed'),
+                'flow-processed.cylc'
```

Comment:
I suppose we should have a `WorkflowFiles.PROCESSED_FLOW_FILE`, but no biggie.

Reply:
Fair-enough: I didn't think it was going to  be re-used.